config/bootstrap.php

```diff
@@ -1,22 +0,0 @@
-<?php
```

Comment:
The bootstrap file was [removed](https://github.com/symfony/symfony-docs/issues/13759) and replaced in symfony 5.1 by a new DotEnv->bootEnv method.